<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steam Controller Jingle Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="browsePushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="convertedJinglesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Converted Jingles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicXmlFilePathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>musicXML File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="chanSourceLeftLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L Channel Source:</string>
    </property>
   </widget>
   <widget class="QLabel" name="chanChordLeftLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L Channel Chord Sel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scserialPortLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SC Serial Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveJinglesPushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>390</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to EEPROM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chanSourceLeftComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="chanChordLeftComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="chanSourceRightComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chanSourceRightLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Channel Source:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chanChordRightComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chanChordRightLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Channel Chord Sel:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playJinglePushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>330</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Jingle</string>
    </property>
   </widget>
   <widget class="QToolButton" name="delJingleToolButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="mvJingleUpToolButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="mvJingleDownToolButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="musicXmlPathLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="jingleListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string extracomment="beep beep"/>
    </property>
   </widget>
   <widget class="QComboBox" name="startMeasComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="endMeasComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="startMeasLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Measure:</string>
    </property>
   </widget>
   <widget class="QLabel" name="endMeasLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End Measure:</string>
    </property>
   </widget>
   <widget class="QLabel" name="octaveAdjustLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Octave Adjust:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpmLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BPM:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bpmLineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="octaveAdjustLineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="memUsageProgressBar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="memUsageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jingle Mem Used:</string>
    </property>
   </widget>
   <widget class="QLabel" name="memUsageCurrBytesLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0/1024 bytes used</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearJinglesPushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>360</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear EEPROM</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteIntensityLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note Intensity:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="noteIntensityLineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
